### PR TITLE
Add channel mask

### PR DESCRIPTION
**Pull request by haoxusci**

### CODE REVIEW

File: src/SingleCellAnnotator.imjoy.html [hunk 1]
```diff
@@ -473,9 +479,18 @@
       });
       api.alert(JSON.stringify(blob))
     },
+    // go to nextImage after select yes/no.
+    async selectLocation(judgement){
+      if (judgement===true){
+      this.current_location_selected = true;}
+      if (judgement===false){
```
Comment (oeway) [inline src/SingleCellAnnotator.imjoy.html]: `judgement` ==> `selected`
and you should use `else` for the other case.

File: src/SingleCellAnnotator.imjoy.html [hunk 2]
```diff
@@ -416,6 +421,7 @@
       const response = await fetch(url_post, {
           method: 'POST',
           mode: 'cors',
+          credentials: 'include',
```
Comment (oeway) [inline src/SingleCellAnnotator.imjoy.html]: Note, starting now, we don't use credentials, please make sure you integrate the changes in #28 : https://github.com/CellProfiling/cellpro-imjoy-plugins/pull/28/files#diff-41fa0174fd16cdd8ccbab0a82b5d597dR212

File: src/SingleCellAnnotator.imjoy.html (hunk 1 above)
Comment (haoxusci) [reply src/SingleCellAnnotator.imjoy.html]: done

File: src/SingleCellAnnotator.imjoy.html (hunk 2 above)
Comment (haoxusci) [reply src/SingleCellAnnotator.imjoy.html:429]: it cannot work with post for annotation, if dont include that. As I ve tested with casper

**Comment by oeway** [comment]
please integrate changes from #28

File: src/SingleCellAnnotator.imjoy.html (hunk 2 above)
Comment (haoxusci) [reply src/SingleCellAnnotator.imjoy.html]: got it

**Pull request merged**